Compiling Kaluga fails with "the trait `std::error::Error` is not implemented for `ili9341::DisplayError`"

Steps to reproduce
Checkout code and configure for ESP32S2.
`cargo build --features kaluga`

Expected:
Code compiles successfully

Observed:
```
error[E0277]: the trait bound `ili9341::DisplayError: std::error::Error` is not satisfied
   --> src/main.rs:860:10
    |
860 |         )?;
    |          ^ the trait `std::error::Error` is not implemented for `ili9341::DisplayError`
    |
    = note: required because of the requirements on the impl of `From<ili9341::DisplayError>` for `anyhow::Error`
    = note: required because of the requirements on the impl of `FromResidual<std::result::Result<Infallible, ili9341::DisplayError>>` for `std::result::Result<(), anyhow::Error>`

For more information about this error, try `rustc --explain E0277`.
```

Workaround:
Replace on line 860 the `)?;` with `).unwrap();`

Code: commit 9b4da900a4ed5a3c2af8eb6bfd3817b6a3c12ea4 from 2022-05-23.

--- a/src/main.rs
+++ b/src/main.rs
@@ -768,9 +768,7 @@ fn ttgo_hello_world(
 ) -> Result<()> {
     info!("About to initialize the TTGO ST7789 LED driver");
 
-    let config = <spi::config::Config as Default>::default()
-        .write_only(true)
-        .baudrate(80.MHz().into());
+    let config = <spi::config::Config as Default>::default().baudrate(26.MHz().into());
 
     let mut backlight = backlight.into_output()?;
     backlight.set_high()?;
@@ -857,7 +855,8 @@ fn kaluga_hello_world(
             &mut delay::Ets,
             KalugaOrientation::Landscape,
             ili9341::DisplaySize240x320,
-        )?;
+        )
+        .map_err(|e| anyhow::anyhow!("Display error: {:?}", e))?;
 
         led_draw(&mut display).map_err(|e| anyhow::anyhow!("Display error: {:?}", e))
     } else {

